packages/utils/src/address.ts

```diff
@@ -33,3 +33,56 @@ export const isValidPrivateKey = (privateKey: string) =>
 export { isValidAddress, isValidChecksumAddress, toChecksumAddress } from 'ethereumjs-util'
 
 export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+export const findAddressIndex = (address: Address, addresses: Address[]) =>
+  addresses.findIndex((x) => eqAddress(x, address))
+
+export const mapAddressListOnto = (oldAddress: Address[], newAddress: Address[]) => {
```

Comment:
what does this do?

Reply:
Added some comments. This is only used by a slasher bot in a different PR that we're currently choosing not to publish: https://github.com/celo-org/celo-monorepo/compare/jfoutts/slasher-bot#diff-d81498e3cf752be28e3ca22b0b6ab510R84 The intended purpose is mapping the index of a validator in the current epoch's validator set, to the corresponding index in the next epoch's validator set, if any.